### ISSUE TITLE
A character icon is not displayed (OUTTRIGGER Dreamcast) 

### ISSUE THREAD

**Issue by Biryuk666** [flyinghead/flycast#564]
Hello! I wrote this in retroarch, but I was sent here

On the level selection screen in versus mode, the player icon is not displayed

Here's a link to the screenshot:
[Character icon](https://cloud.mail.ru/public/dFXk/gNzNB11dE)

All you need to do to see it, you need to choose vs mode => choose the character => give him a name and when loading the screen with stages, in the upper left corner of the screen there will be a regular colored square without a character icon

Game Version: Any Dreamcast Version

Emulator: RetroArch (latest steam version)
Core: Flycast 

OS: Windows 10 x64

System specification: amd ryzen 2700x; amd radeon rx480 8gb; RAM 2x 8GB DDR4 2300MHz

**Comment by flyinghead** [comment]
I can confirm the issue but I haven't found the cause of the problem.

**Comment by Biryuk666** [comment]
> confirm

Hello! Previously, redream fixed this bug, but another one appeared. They made a translucent background and an icon appeared. It seems to me that the character icon is in the background, behind the background square. The redream team did so that the icon appeared, but because of this began problems with the HUD

**Comment by flyinghead** [comment]
Yes, I pretty much came to the same conclusion: I can't find what isn't correctly emulated and any change would impact other games or other graphic elements.

**Comment by kihato** [comment]
Other game examples.

Capcom vs. SNK - Millennium Fight 2000 Pro
![CvsSMF2Kpro](https://github.com/flyinghead/flycast/assets/14366233/4fc47120-66c9-40be-ba57-1e4d9cc3a86d)
Test Drive 6
![TestDrive6](https://github.com/flyinghead/flycast/assets/14366233/81bb6cb1-830a-46eb-b8bd-40800e095d1b)
